### ISSUE TITLE
Document the .fbx interface

### ISSUE THREAD

**Issue by plicease** [PerlFFI/FFI-Platypus#221]
You can specify cflags, libs or aliens via the `.fbx` file in the `ffi` directory.  This hasn't been well documented.  If at all, and this should be rectified.  The `[AlienBuild]` plugin ought to have a mention of it as well.

**Comment by plicease** [comment]
The TL;DR (or rather didn't exist ; didn't read), is that to use an alien with bundled code in an ffi directory you can place a `ffi/something.fbx` file in your dist (the basename "something" here can be whatever you choose) with content like this:

```perl
use strict;
use warnings;

{
  verbose => 1,
  alien  => 'Alien::libfoo',
}
```

It's perl, so you can write dynamic or platform specific flags that get passed into `FFI::Build`.

If you need to specify a flags with some relative paths, say you had an include directory, you can use  `$DIR`:

```perl
use strict;
use warnings;

our $DIR;

{
  verbose => 1,
  cflags_I => "-I$DIR/include",
}
```

**Comment by plicease** [comment]
This was addressed with #306
